docs/source/create_channel/create_channel_overview.rst

```diff
@@ -0,0 +1,31 @@
+Creating a channel
+==================
+
+In order to create and transfer assets on a Hyperledger Fabric network, an
+organization needs to join a channel. Channels are a private layer of communication
+between specific organizations. Each channel consists of a separate ledger
+that can only be read and written to by channel members, who are allowed to
+join their peers to the channel and receive new blocks of transactions from the
+ordering service. While the peers, nodes, and Certificate Authorities form the
+physical infrastructure of the network, channels are the process by which
+organizations connect with each other and interact. Channels are invisible to
+other members of the network, making them the most powerful tool for privacy in
+a blockchain consortium.
```

Comment:
Channels are invisible to
other members of the network, making them the most powerful tool for privacy in
a blockchain consortium.

`most powerful tools` seems strange to me. I would maybe say:
Channels are invisible to
other members of the network, making them a powerful basis for privacy in
a blockchain consortium.

Reply:
I should have removed that. Private data is more powerful a privacy tool.